lib/converter.js

```diff
@@ -104,7 +104,9 @@ async function convert (processor, inputFile, options, timings, preview) {
       // capture console output
       page.on('console', msg => consoleLog(msg))
     }
-    await page.goto(`file://${tempFile}`, { waitUntil: 'networkidle2' })
+    await page.goto(`file://${tempFile}`, { waitUntil: 'networkidle0' })
+    const watchDog = page.waitForFunction('window.status === undefined || window.status === "ready"')
```

Comment:
`window.status` is probably too generic: https://developer.mozilla.org/en-US/docs/Web/API/Window/status
I think we should use a reserved namespace like `window.AsciidoctorPDF.status`.

Reply:
Or maybe it was intended to set the text on the status bar?

Reply:
will use `AsciidoctorPDF`; using just `status` was my mistake.